doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1113 @@
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added new ``west ncs-sbom`` command that generates :ref:`Software Bill of Materials <west_sbom>`.
+* Added :ref:`bt_fast_pair_provision_script`.
+
+* :ref:`partition_manager`:
+
+  * Added the :file:`ncs/nrf/subsys/partition_manager/pm.yml.pgps` file.
+  * Added the :file:`ncs/nrf/subsys/partition_manager/pm.yml.emds` file.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``e86f575f68fdac2cab1898e0a893c8c6d8fd0fa1``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` to attach the dependencies between application and network core images.
+  This option links the upgrade images in such a way that either both or none is applied.
+* Added support for the Dual-slot execute-in-place (XIP) feature in the |NCS| build system.
+  See the :file:`ncs/nrf/tests/modules/mcuboot/direct_xip` test project for an example of how to leverage this feature in your application.
+* boot_serial: Adapt to Zephyr's new CRC APIs (boot_serial implements the serial recovery),
+* zephyr/boot_serial_extension: use ``BOOT_LOG`` instead of ``LOG_`` (boot_serial_extension implements the serial recovery extension).
+* Loader was reworked so it allows larger minimum flash write size (was 8 B, became 32 B).
+* boot_serial: zephyr: Add optional timeout to enter serial recovery.
+  See :kconfig:option:`CONFIG_BOOT_SERIAL_WAIT_FOR_DFU`.
+* zephyr: Use a smaller SHA-256 implementation.
+* boot_serial: Added support for the echo command.
+  See :kconfig:option:`CONFIG_BOOT_MGMT_ECHO`.
+* single loader: fixed image decryption for any SoC flash of the pages with size not fitting in 1024 B.
+* Removed deprecated ``DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL``.
+* Fixed usage of ``CONFIG_LOG_IMMEDIATE``.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``53fbf40227de087423620822feedde6c98f3d631``, plus some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline 53fbf40227 ^45ef0d2
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^53fbf40227
+
+The current |NCS| main branch is based on revision ``53fbf40227`` of Zephyr.
+
+cddl-gen/zcbor
```

Comment:
This section is slightly "outdated", since after this was done, zcbor was also added to Zephyr, and subsequently brought into NCS via upmerge, but this shift should have been invisible to users. We could add a note that "zcbor is now provided via Zephyr instead of directly via NCS" or something.

Reply:
added